File: Python/retangulo.py
import math

base = float(input("base do retângulo: "))
altura = float(input("altura do retângulo: "))

area = base * altura
perimetro = 2 * (base + altura)
diagonal = math.sqrt(base ** 2 + altura ** 2)

print(f"AREA = {area:.4f}")
print(f"PERÍMETRO = {perimetro:.4f}")
print(f"DIAGONAL = {diagonal:.4f}")

File: C/retangulo.c
#include <stdio.h> 
#include <math.h>

int main()
{
    double base, altura, area, perimetro, diagonal;

    printf("base do retângulo: ");
    scanf("%lf", &base);

    printf("altura do retângulo: ");
    scanf("%lf", &altura);

    area = base * altura;
    perimetro = 2 * (base + 2 * altura);
    diagonal = sqrt(pow(base, 2.0) + pow(altura, 2.0));

    printf("AREA = %.4lf\n", area);
    printf("PERÍMETRO = %.4lf\n", perimetro);
    printf("DIAGONAL = %.4lf\n", diagonal);
    
    return 0;
}
